### PR TITLE
Remove the deprecation warning in React 15

### PR DESCRIPTION
**Pull request by letrunghieu**
Remove the warning message when using with React 15.2.1 as mentioned in #80

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -91,9 +91,29 @@ module.exports = React.createClass({
 		});
 	},
 	render: function () {
+		var ignoredAttrs = [
+			"alwaysShowCalendars", "applyClass", "autoApply",
```
Comment (olegsklyarov) [inline lib/index.js]: It's better to get all this items from lib/get-options.js to avoid code duplication.

Comment (letrunghieu) [inline lib/index.js]: Thank you for pointing out this. I updated the code.

**Comment by Zephir77167** [comment]
Nice, we need this! :+1:

**Comment by skratchdot** [comment]
Sorry I sat on this for so long @letrunghieu. I just pulled in some changes in pr #85, and now I can't pull this in due to merge conflicts.

**Comment by skratchdot** [comment]
This should be fixed in the latest versions > 3.2.0.  Can you please re-open this bug if it's not the case?

**Comment by letrunghieu** [comment]
I think this should be fixed with the PR #85, too. I have not tried with the new version yet. I'll re-open it if it's not the case.

**Pull request closed without merging**